version.json

```
@@ -12,6 +12,7 @@
   // Adding additional ref specs here may result in builds of different commits producing the same version numbers.
   // Ensure you know what you are doing before adding any other ref specs here.
   "publicReleaseRefSpec": [
-    "^refs/heads/master$"
+    "^refs/heads/master$",
+    "^refs/heads/v\\d+\\.\\d+$"
```

I believe this should be tags instead of heads, and that the regex should allow for more than just major and minor version numbers
```suggestion
    "^refs/tags/v\\d+\\.\\d+"
```

---

Yes good point